Casks/s/sqlitestudio.rb

```diff
@@ -1,6 +1,6 @@
 cask "sqlitestudio" do
-  version "3.4.4"
-  sha256 "bd1bf5cd0e442b867ef9417e6c849d7b9f4d38f4305804c4b9d58d905092d8ef"
+  version "3.4.5"
+  sha256 "ee9197ab85802156875ffb7768695c4cd4119c7126302244e3b156484c6326a7"
 
   url "https://github.com/pawelsalawa/sqlitestudio/releases/download/#{version}/SQLiteStudio-#{version}.dmg",
```

Comment:
```suggestion
  url "https://github.com/pawelsalawa/sqlitestudio/releases/download/#{version}/sqlitestudio-#{version}.dmg",
```

It looks like the upstream filename is `sqlitestudio-3.4.5.dmg` but `SQLiteStudio-3.4.5.dmg` works because GitHub seems to find the file regardless of letter case. Though the existing URL works, there's maybe something to be said for matching the upstream filename format.